deepchecks/__init__.py

```diff
@@ -9,13 +9,19 @@
 # ----------------------------------------------------------------------------
 #
 """Deepchecks."""
-import os
-import pathlib
 import sys
 import types
 import warnings
 from importlib._bootstrap import _init_module_attrs
 
+try:
+    from importlib.metadata import version
+except ImportError:
+    from importlib_metadata import version
+    
+# NOTE: it is here, before other import, in order to omit circular import error
+__version__ = version('deepchecks')
```

Comment:
hm/

for users, probably not, because `version` function expects that the given package was installed as a library (via pip)

it might fail for you if you have not done `pip install -e .` and expects that `deepchecks` will be in the `sys.path`